hypercx-nfv/deb-package/hypercx-nfv-1.0.0/DEBIAN/postinst

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+OPENVPN_STATUS=$(systemctl status openvpn@server | grep 'Active:' | awk '{print $3}' | grep 'running' | wc -l)
```

Comment:
todas estas comprobaciones estan mal. quitalas

Reply:
ademas, para comprobar el estado de un servicio se usa systemctl is-enabled, pero aca no hace falta